lms/djangoapps/grades/tasks.py

```diff
@@ -189,7 +188,8 @@ def _update_subsection_grades(
                         subsection_grade=subsection_grade,
                     )
 
-        except DatabaseError as exc:
+        except Exception as exc:   # pylint: disable=broad-except
+            log.info("_update_subsection_grades unexpected failure, retrying. Error: {}".format(repr(exc)))
```

Comment:
Should we specifically catch expected errors (DatabaseError / ValidationError), and not log those?